composer.json

```diff
@@ -101,12 +101,23 @@
     "cache/integration-tests": "^0.16.0",
     "codeception/codeception": "~2.4.5",
     "phpstan/phpstan": "^0.12",
-    "phpstan/phpstan-symfony": "^0.12"
+    "phpstan/phpstan-symfony": "^0.12",
+    "heidelpay/heidelpay-php": "^1.2.5.1",
```

Comment:
Imho those dependencies should stay optional.

Reply:
Its only "require-dev". Is only installed if the composer is run directly on the package (root-only). See https://getcomposer.org/doc/04-schema.md#require-dev

Reply:
Oh, sorry, missed that detail in the diff view 😊 Thanks for clarifying!

Reply:
I added this packages also to `suggest`. So the user see, which packages can be added to increase functionality.